packages/payment/src/models/payment.ts

```diff
@@ -30,11 +35,11 @@ export default class Payment {
   @PrimaryKey({ columnType: "text" })
   id: string
 
-  @Property({
-    columnType: "numeric",
-    serializer: Number,
-  })
-  amount: number
+  @MikroOrmBigNumberProperty()
+  amount: BigNumber | number
```

Comment:
tests are failing because we end up with the amounts as strings

Reply:
I ll have a look @olivermrbl

Reply:
@fPolic @olivermrbl you can have a look [here](https://github.com/medusajs/medusa/pull/6509/commits/f86bf38934a466530a558d3e117873e50f9b13bf)

Reply:
Looks good @adrien2p 👍 

Also, should we define the amount as `number` only? From the current implementation of the decorator, the field is always set as bignumber.numeric so it will never be a BigNumber?